config_src/nuopc_driver/mom_cap_methods.F90

```diff
@@ -942,16 +939,16 @@ subroutine field_getfldptr(field, fldptr1, fldptr2, rank, abort, rc)
 
 end subroutine field_getfldptr
 
-logical function chkerr(rc, line, file)
+logical function ChkErr(rc, line, file)
```

Comment:
Thanks, @jiandewang. My last request: please document this function and its arguments.

Reply:
@jiandewang, documenting the function ChkErr and its arguments is the last thing that I think needs to be done before this PR is ready to go.